network/receiver.go

```diff
@@ -51,6 +51,7 @@ func handleReceiving(log zerolog.Logger, wg *sync.WaitGroup, cfg *Config, rep re
 			continue
 		}
 		input <- msg
+		eventMgr.Received(address, msg)
```

Comment:
It really shouldn't call this here. This would mean that we would send all messages to the subscriber, including the signaling messages that should be internal to the library. Custom messages would even be sent to the subscriber twice, once in the receiver and once in the processor.

Reply:
Good point, I didn't realize that we don't want to publish ping/pong and other internal events.